### PR TITLE
Update documentation-style-guide.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Adding information about notes, tips, and so on to the style guide. Feel free to add on.

### CODE REVIEW

File: contribute/style-guides/documentation-style-guide.md [hunk 1]
```diff
@@ -115,6 +115,34 @@ When possible, use the exact title of the page or section you are linking to as
...
+
+Notes are for additional information that the user should be extra aware of. For example:
+
+> **Note:** This page describes a feature for Grafana 7.0 beta.
+
+#### Tips
+
+Tips are alternate or more efficient ways of doing things. Rarely used.
+
+#### Cautions
```
Comment (achatterjee-grafana) [inline contribute/style-guides/documentation-style-guide.md:136]: Should be Caution?

File: contribute/style-guides/documentation-style-guide.md [hunk 2]
```diff
@@ -115,6 +115,34 @@ When possible, use the exact title of the page or section you are linking to as
...
+
+Cautions warn the user that they should proceed with caution. Use cautions to emphasize the potential downside of a course of action.
+
+#### Warnings
```
Comment (achatterjee-grafana) [inline contribute/style-guides/documentation-style-guide.md:140]: Should be Warning?

Comment (achatterjee-grafana) [inline contribute/style-guides/documentation-style-guide.md:140]: Similarly, Warning

File: contribute/style-guides/documentation-style-guide.md (hunk 1 above)
Comment (oddlittlebird) [reply contribute/style-guides/documentation-style-guide.md:136]: No. I made Cautions and Warnings plural to maintain parellelism with Notes and Tips.

**Comment by achatterjee-grafana** [comment]
Got you. You can add examples for "Tip" and "Caution" later, as you see fit.
Approving.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**